public/app/features/alerting/unified/components/import-to-gma/NamespaceAndGroupFilter.tsx

```diff
@@ -0,0 +1,121 @@
+import { css } from '@emotion/css';
+import { useMemo } from 'react';
+import { Controller, useFormContext } from 'react-hook-form';
+
+import { GrafanaTheme2, SelectableValue } from '@grafana/data';
+import { Field, Stack, VirtualizedSelect, useStyles2 } from '@grafana/ui';
+import { t } from 'app/core/internationalization';
+
+import { useGetNameSpacesByDatasourceName } from '../rule-editor/useAlertRuleSuggestions';
+
+import { ImportFormValues } from './ImportFromDSRules';
+
+interface Props {
+  rulesSourceName: string;
+}
+
+export const NamespaceAndGroupFilter = ({ rulesSourceName }: Props) => {
+  const {
+    control,
+    watch,
+    formState: { errors },
+    setValue,
+  } = useFormContext<ImportFormValues>();
+
+  const style = useStyles2(getStyle);
+  const namespace = watch('namespace');
+  const { namespaceGroups, isLoading } = useGetNameSpacesByDatasourceName(rulesSourceName);
+
+  const namespaceOptions: Array<SelectableValue<string>> = useMemo(
+    () =>
+      Array.from(namespaceGroups.keys()).map((namespace) => ({
+        label: namespace,
+        value: namespace,
+      })),
+    [namespaceGroups]
+  );
+
+  const groupOptions: Array<SelectableValue<string>> = useMemo(
+    () => (namespace && namespaceGroups.get(namespace)?.map((group) => ({ label: group, value: group }))) || [],
+    [namespace, namespaceGroups]
+  );
+
+  return (
+    <Stack direction="row" gap={2}>
+      <Field
+        data-testid="namespace-picker"
+        label={t('alerting.import-to-gma.namespace.label', 'Namespace')}
+        // Disable translations as we don't intend to use this dropdown longterm,
+        // so avoiding us adding translations for the sake of it
+        // eslint-disable-next-line @grafana/no-untranslated-strings
+        description={t('alerting.import-to-gma.namespace.description', 'Type to search for an existing namespace')}
```

Comment:
We ended up translating these here - I guess we do expect this approach to remain for a little longer? The other component had the rule commented out because ideally we want to make the namespace/group selection look more like the GMA approach

Reply:
we have already translated..removing the comment